src/utils.ts

```diff
@@ -529,6 +541,20 @@ export class Utils {
     (target || e.target).dispatchEvent(simulatedEvent);
   }
 
+  public static getNewXYPosition(event: MouseEvent, element: GridItemHTMLElement) {
+    const containerElement = Utils.getContainerElement(element.parentElement);
+    const containerBoundingClientRect = containerElement.getBoundingClientRect();
+    const boundingClientRect = element.getBoundingClientRect();
+    const scaleX = boundingClientRect.width / element.offsetWidth;
+    const scaleY = boundingClientRect.height / element.offsetHeight;
+
+    // Position the element behind the mouse
+    const node = element.gridstackNode || {};
+    const x = (event.clientX - containerBoundingClientRect.left - (node._originalMousePositionInsideElement?.x || 0)) / scaleX;

```

Comment:
can't have a public Util APi call internal ._originalMousePositionInsideElement var from something, that breaks the internal boundary...